src/test/java/seedu/address/logic/parser/GoToCommandParserTest.java

```diff
@@ -55,7 +59,8 @@ public void parse_emptyArg_throwsParseException() {
     @Test
     public void parse_validArgs_returnsFindCommand() {
```

Comment:
```suggestion
    public void parse_validArgs_returnsGoToCommand() {
```
Oops think there is a little typo here from the past commits